dashboard/test/controllers/activities_controller_test.rb

```diff
@@ -615,8 +613,7 @@ def _test_logged_in_milestone_with_image(async_activity_writes:)
     expected_response = build_expected_response(
       level_source: "http://test.host/c/#{assigns(:level_source).id}",
       save_to_gallery_url: "/gallery"\
-        "?gallery_activity%5Bactivity_id%5D=#{assigns(:activity).id}"\
-        "&gallery_activity%5Blevel_source_id%5D=#{assigns(:level_source).id}"\
+        "?gallery_activity%5Blevel_source_id%5D=#{assigns(:level_source).id}"\
         "&gallery_activity%5Buser_level_id%5D=#{assigns(:user_level).try(:id)}"
```

Comment:
This is generated by `application_controller.rb` (specifically Line 93) see my change to that file in this PR.